miasm2/arch/aarch64/sem.py

```diff
@@ -526,9 +527,27 @@ def msub(arg1, arg2, arg3, arg4):
     arg1 = arg4 - (arg2 * arg3)
 
 
-@sbuild.parse
-def udiv(arg1, arg2, arg3):
-    arg1 = m2_expr.ExprOp('udiv', arg2, arg3)
+def udiv(ir, instr, arg1, arg2, arg3):
```

Comment:
Here, you can still use `sbuild.parse`:
```Python
@sbuild.parse
def udiv(arg1, arg2, arg3):
    if arg3:
        arg1 = m2_expr.ExprOp('udiv', arg2, arg3)
    else:
        exception_flags = m2_expr.ExprInt(EXCEPT_INT_DIV_BY_ZERO, exception_flags.size)
```

(which I personally found more readable)
Do not forget to add `exception_flags` to the SemBuilder namespace (L140):
```Python
+        "exception_flags": exception_flags,
```

Other possible form:
```Python
arg1 = 'udiv'(arg2, arg3)
```